docs/home/howto/CLUSTER.md

```diff
@@ -292,6 +292,35 @@ To setup the cluster do the following steps from the provisioning machine:
     epicli apply -f newcluster.yml
     ```
 
+## Note for Red Hat / CentOS Azure images
```

Comment:
3 times # if this is a subpart of "How to create an Epiphany cluster on a cloud provider"

Reply:
Added subpart